src/components/dashboard/ClaimButton.js

```diff
@@ -1,41 +1,52 @@
-import React from 'react'
+import React, { memo, useCallback, useEffect, useRef } from 'react'
 import { View } from 'react-native'
+import CardFlip from 'react-native-card-flip'
+import Config from '../../config/config'
 import { withStyles } from '../../lib/styles'
 import { weiToGd } from '../../lib/wallet/utils'
 import { CustomButton } from '../common'
 import BigGoodDollar from '../common/view/BigGoodDollar'
 import Text from '../common/view/Text'
 import Section from '../common/layout/Section'
-import { getDesignRelativeWidth } from '../../lib/utils/sizes'
+import { getDesignRelativeHeight, getDesignRelativeWidth } from '../../lib/utils/sizes'
 
-const ButtonAmountToClaim = ({ entitlement, isCitizen, styles }) => (
-  <>
-    <Text color="surface" fontWeight="medium">
-      {`CLAIM YOUR SHARE - `}
-    </Text>
-    <BigGoodDollar
-      number={entitlement}
-      formatter={weiToGd}
-      fontFamily="Roboto"
-      bigNumberProps={{
-        fontFamily: 'Roboto',
-        fontSize: isCitizen ? 36 : 16,
-        color: 'surface',
-        fontWeight: 'medium',
-        lineHeight: 36,
-      }}
-      bigNumberUnitProps={{
-        fontFamily: 'Roboto',
-        fontSize: isCitizen ? 16 : 10,
-        color: 'surface',
-        fontWeight: 'medium',
-        lineHeight: 19,
-        marginVertical: 'auto',
-      }}
-      style={isCitizen ? styles.amountInButtonCenter : styles.amountInButton}
-    />
-  </>
-)
+const ButtonAmountToClaim = ({ entitlement, isCitizen, styles }) =>
+  Config.claimContentPhaseZero ? (
```

Comment:
do not use config here. your kind of design should be defined **on claim screen only**
add some prop e.g. displayNextClaim: boolean and pass it dependently of Config.claimContentPhaseZero **from Claimphase1 component**